### ISSUE TITLE
Metrics of the Analyzer

### ISSUE THREAD

**Issue by juanjolmm** [anchore/anchore-charts#87]
Hello, I have just installed the Anchore Helm chart and I am very interested in having access to the metrics.
I see that we can have access to the metrics of each component by accessing the /metrics path.
In my case, I have deployed Anchore using an ALB ingress, everything is working fine, but I need a service for the analyzer, that does not exist.
If I am not wrong the only way I have to access the analyzer metrics is to create an extra service.
Am I missing something?

Thank you very much in advance.

**Comment by hbermu** [comment]
Thanks for the Service yaml @ArjonBu! I have more than 1 analyzer, so this solution can't fix my problem at all. 
Thank you very much.

**Comment by ArjonBu** [comment]
@hbermu Yes, it does. I am afraid you might not know how kubernetes services work.
